internal/pkg/describe/status.go

```diff
@@ -401,3 +435,52 @@ func statusColor(status string) string {
 		return color.Red.Sprint(status)
 	}
 }
+
+type targetHealthStatus struct {
+	TargetHealth elbv2.TargetHealth `json:""`
+	TaskID       string             `json:"taskID"`
+}
+
+func (s *targetHealthStatus) relateToTask(taskID string) {
+	s.TaskID = taskID
+}
+
+func (s *targetHealthStatus) humanString() string {
+	relatedTask := "-"
```

Comment:
If i remember correctly over 90% ECS users only have one target group per service (plus we don't natively support MTG in copilot). We probably don't need to handle duplicated taskID to be "-".